frontend/src/components/VariantListPage/VariantsTable.tsx

```diff
@@ -723,6 +732,20 @@ const VariantsTable: FC<VariantsTableProps> = ({
             </Checkbox>
           </Td>
         )}
+        {includeTagColumn && (
+          <Td
+            sx={{
+              height: `${ROW_HEIGHT}px`,
+              alignContent: "center",
+            }}
+          >
```

Comment:
Here's where you could define the width of the column, a la the other columns, to prevent the growing and shrinking of the column when tags get added, removed, or name changed, per Matt's comment in standup yesterday.